core-library/src/main/java/org/silverpeas/core/admin/space/dao/SpaceDAO.java

```diff
@@ -143,6 +143,7 @@ private SpaceInstLight fetchSpace(ResultSet rs) throws SQLException {
     space.setName(rs.getString("name"));
     space.setOrderNum(rs.getInt("orderNum"));
     space.setLook(rs.getString("look"));
+    space.setStatus(rs.getString("spaceStatus"));
```

Comment:
see my answer to the referred feedback. This modification is required because without it, this is a bug: the status has to be fetched when loading a space from the database. So, even if `RemovedSpaceAndComponentInstanceChecker`  is used, the status of the space has to be fetched along with the others data.